Objects/typeobject.c

```diff
@@ -4540,6 +4578,13 @@ PyType_GetQualName(PyTypeObject *type)
     return type_qualname(type, NULL);
 }
 
+PyObject *
+PyType_GetFullyQualifiedName(PyTypeObject *type)
+{
+    return type_get_fullyqualname(type, NULL);
```

Comment:
```suggestion
    return type_fullyqualname(type, 0);
```
No need to call the intermediate function

Reply:
I reorganized the calls. The special function with ``int is_repr`` parameter now gets a ``_impl`` suffix. The getter losts `get_` in its name to look alike type_name() and type_qualname() getters. It now calls PyType_GetFullyQualifiedName(). Does it look better to you?